transactions_test.go

```diff
@@ -573,14 +576,14 @@ func TestTransactions_Err_FraudCard(t *testing.T) {
 		t.Fatalf("error occurred making API call. Err: %s", err)
 	} else if r.IsOK() {
 		t.Fatal("expected create fraudulent transaction to return error")
-	} else if !reflect.DeepEqual(r.TransactionError, &recurly.TransactionError{
+	} else if !reflect.DeepEqual(transaction.TransactionError, &recurly.TransactionError{
```

Comment:
Since we are now assigning `resp.Transaction` struct to `transaction` on error, please add a `reflect.DeepEqual` of `transaction` against `*recurly.Transaction` so we can ensure the entire transaction is returned with the embedded TransactionError (not just an empty transaction with only the `TransactionError` field)